README.md

```diff
@@ -208,59 +208,85 @@ plaidPromise.then(successResponse => {
 });
 ```
 
-Plaid API methods that may return an MFA response pass an array of responses
-to the next promise. The first element of the array is the `mfaResponse`, and
-the second element is the `successResponse`. You can use destructuring to
-improve readability.
-
 For example:
 
 ```javascript
+'use strict';
+
+const bodyParser = require('body-parser');
+const express = require('express');
 const plaid = require('plaid');
 
-const plaidClient = new plaid.Client(CLIENT_ID, SECRET, PUBLIC_KEY, plaid.environments.sandbox);
+const plaidClient = new plaid.Client(
+  process.env.PLAID_CLIENT_ID,
+  process.env.PLAID_SECRET,
+  process.env.PUBLIC_KEY,
+  plaid.environments.sandbox
+);
+
+const app = express();
+const port = process.env.PORT || 3000;
+
+app.use(bodyParser.urlencoded({
+    extended: true
+}));
+app.use(bodyParser.json());
+
+app.post('/plaid_exchange', (req, res) => {
+  var public_token = req.body.public_token;
+
+  plaidClient.exchangePublicToken(public_token).then(res => {
+    const access_token = res.access_token;
+
+    plaidClient.getAccounts(access_token).then(res => {
+      console.log(res.accounts);
+    });
+  }).catch(err => {
+    // If error_type is null, it indicates a network or runtime error.
```

Comment:
I believe this is the proper way to interact with the node client errors. @michaelckelly does this seem right to you? 

Maybe we should export some sort of `isPlaidError` function from this package to make disambiguating plaid errors from runtime/network errors easier for clients.